### DISCUSSION TITLE
Lost email access password reset

### DISCUSSION THREAD

**Discussion by bayareaunicorn** [community/community#79309] [Other Feature Feedback, Questions, & Ideas] [answered]
Had an SSD failure using tutanota as email provider and use keepass as a password manager. I have the GitHub app here and am logged in and also have 2fa totp codes which I can still access. How can I add an alternative email from the app or update my password?

**Comment by taaake** [comment]
Try to find a way to add an alternative e-mail address. You need to find a section where you can manage the e-mail addresses associated with your account

**Reply by bayareaunicorn** [reply]
Where in the app?

**Comment by bayareaunicorn** [comment, marked as answer]
I got back into my app but it would be great to have more setting options within the Android App.
